src/doc/man/cargo-vendor.md

```diff
@@ -21,6 +21,12 @@ stdout after `cargo vendor` completes the vendoring process.
 You will need to add or redirect it to your Cargo configuration file,
 which is usually `.cargo/config.toml` locally for the current package.
 
+Cargo treats vendored sources as read-only as it does to registry and git sources.  
```

Comment:
Just notice there are some trailing spaces. Not sure if it is intentionally done but they do render to a linebreak here. Otherwise looks good to me.

```suggestion
Cargo treats vendored sources as read-only as it does to registry and git sources.
```

Reply:
More thanks, modified